lib/enops/support/aws_cli_credential_process

```diff
@@ -47,6 +47,11 @@ def run(*args, nil_on_error: true)
 end
 
 def read_password(msg = 'Password')
+  if (enops_ykman_account = ENV["ENOPS_YUBIKEY_ACCOUNT"])
+    code = run(%W[ykman oath accounts code #{enops_ykman_account} -s])
+    return code if code
+  end
```

Comment:
`read_password` is intended to be generic. We could rename to `get_mfa_token` or similar but maybe make this a `get_ykman_code` method and chain it on the `mfa_token = …` line?